main.cpp

```diff
@@ -194,17 +238,27 @@ struct MidiController
 
 void MidiController::controlSynth(int midiMessage)
 {
-
+    if (midiMessage == 74)
+    {
+        std::cout << "cut off adjusting\n.";
+    }
 }
 
 void MidiController::adjustVolume(int midiMessage)
 {
-
+    bool midiVolume;
+    if (midiMessage)
+    
+    {
+        midiVolume = true;
+    }
 }
```

Comment:
- [ ] clean up the empty blank lines
- [ ] you need to compare `midiMessage` with something in that `if()` expression.  What you're doing now is saying "if midi message is any value other than 0, midiVolume is true"
- [ ] you need to initialize your midiVolume variable.